packages/sn-controls-react/src/fieldcontrols/CheckboxGroup.tsx

```diff
@@ -0,0 +1,161 @@
+/**
+ * @module FieldControls
+ */
+import React, { Component } from 'react'
+
+import Checkbox from '@material-ui/core/Checkbox'
+import FormControl from '@material-ui/core/FormControl'
+import FormControlLabel from '@material-ui/core/FormControlLabel'
+import FormGroup from '@material-ui/core/FormGroup'
+import FormHelperText from '@material-ui/core/FormHelperText'
+import FormLabel from '@material-ui/core/FormLabel'
+import TextField from '@material-ui/core/TextField'
+import { ChoiceFieldSetting } from '@sensenet/default-content-types'
+import { ReactClientFieldSetting } from './ClientFieldSetting'
+
+// TODO(Zoli): Review this. It's value should be determind by the allowmultiple setting
```

Comment:
@B3zo0 I hope you've checked this 😃

Reply:
Good catch. I've reviewed and reworked this component.